tap_ms_graph/streams/drive_items.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, Iterator, List
+from singer import get_logger
+from tap_ms_graph.streams.abstracts import FullTableStream
+
+LOGGER = get_logger()
+
+
+class DriveItems(FullTableStream):
+    tap_stream_id = "drive_items"
+    key_properties = ["id"]
```

Comment:
To map child record with parent record we must insert parent id in key_properties.

```suggestion
    key_properties = ["id", "user_id"]
```